app/backend/src/couchers/tasks.py

```diff
@@ -182,8 +182,8 @@ def send_friend_request_email(friend_relationship):
         },
     )
 
-
-def send_host_reference_email(reference, both_written):
+#called in references.py
```

Comment:
No need for this comment, one can easily find this out by searching through the codebase

Reply:
Removed